types/encoding.go

```diff
@@ -9,6 +9,10 @@ import (
 	"github.com/0xPolygon/polygon-edge/helper/hex"
 )
 
+// ParseUint64orHex parses the given string as uint64 in hex
+// It should go to the common package from the logical perspective
+// as well as avoiding cycle imports.
+// DEPRECATED. Use common.ParseUint64orHex.
 func ParseUint64orHex(val *string) (uint64, error) {
```

Comment:
I hope we are about to remove this function through some of the follow-up PRs.

Reply:
Yeah, we will remove it